test/qa/lib/daemon.py

```diff
@@ -1,20 +1,19 @@
 """Functions to make it easier to interact with nordvpnd."""
 import glob
-import logging
 import os
 import socket
 import time
 
 import sh
 
-from . import ssh
+from . import logging, ssh
 
 
 def _rewrite_log_path():
     project_root = os.environ["WORKDIR"].replace("/", "\\/")
     pattern = f"s/^LOGFILE=.*/LOGFILE={project_root}\\/dist\\/logs\\/daemon.log/"
-    #sh.sudo.sed("-i", pattern, "/etc/init.d/nordvpn")
-    os.popen(f"sudo sed -i {pattern} /etc/init.d/nordvpn").read()
+    sh.sudo.sed("-i", pattern, "/etc/init.d/nordvpn")
+    #os.popen(f"sudo sed -i {pattern} /etc/init.d/nordvpn").read()
```

Comment:
Did this just stop working? So we are switching to a different way of doing the same thing?

Reply:
Originally there was `sh.sudo.sed` but not so long time ago it was replaced with `os.popen` - it was part of snap adoption, but that change was unnecessary, because this function runs only in docker without `systemd`